Exercise_4.java

```diff
@@ -1,19 +1,74 @@
-class MergeSort 
+package com.sthirty.precoursetwo.prblmfour;
+
+class Exercise_4 

```

Comment:
Good Job! with implementations and code comments. Please add time and space complexities for your implementations.